src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo,
+  setSelected: (arg: Todo | null) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo, setSelected,
+}) => {
+  const [loader, setLoader] = useState(false);
```

Comment:
```suggestion
  const [loader, setLoader] = useState(true);
```
to fix blinking when open the modal